### ISSUE TITLE
Update autograph calls to send the right parameters for recommended extensions

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons#6522]
Autograph is going to be updated to provide support for adding to the signature of an extension so that Firefox will know that a curated extension is going to be used.

Once that support is ready and we have already done the backend work to support the identification of recommended add-ons we can then conditionally send different params to autograph to request the correct signature attribution.

https://bugzilla.mozilla.org/show_bug.cgi?id=1548647
